tools/flaky_test_monitor/common/level1.go

```diff
@@ -31,14 +32,17 @@ type Level1TestResult struct {
 	CommitSha  string    `json:"commit_sha"`
 	CommitDate time.Time `json:"commit_date"`
 	JobRunDate time.Time `json:"job_run_date"`
+	RunID      string    `json:"run_id"`
 
 	// test specific data
-	Test    string `json:"test"`
-	Package string `json:"package"`
-	Output  []struct {
-		Item string `json:"item"`
-	} `json:"output"`
-	Result   string  `json:"result"`
-	Elapsed  float32 `json:"elapsed"`
-	NoResult bool    `json:"no_result"`
+	Test      string   `json:"test"`
+	Package   string   `json:"package"`
+	Output    []string `json:"output,omitempty"`
+	Elapsed   float64  `json:"elapsed"`
+	Pass      int      `json:"pass"`
+	Exception int      `json:"exception"`
+
+	Action string `json:"-"`
 }
+
+type Level1Summary []Level1TestResult
```

Comment:
I like this change. So we don't need to wrap all the level 1 test results inside a struct but just treat the list of results as a separate type instead